### PR TITLE
Fixed issues #13 #14  #16 

### PR DESCRIPTION
**Pull request by sebjimenez**
# Purpose or Intent

The goal of this PR is to fix the following issues:

1. Issue #13  (**Go Back button** for show shift view) 
2. Issue #14 (**Cancel button** for edit shift view) 
3. Issue #16 (**Redirect** after update shift's info)

For more information feel free to access the links to the mentioned issues.

### CODE REVIEW

**Comment by sebjimenez** [comment]
This is a view that covers the new changes to be merged. @agarciavera Please download the branch an provide feedback.

issue2

**Comment by agarciavera** [comment]
It's working sucessfully. LGTM

**Comment by sebjimenez** [comment]
Ready to merge. @sergio-ocon

File: config/locales/en.yml [hunk 1]
```diff
@@ -57,7 +57,7 @@ en:
     reports:                "reports"
     dashboard:              "dashboard"
     models:
-      errors_prohibited_saving: "prohibited this %{model} from being saved" 
+      errors_prohibited_saving: "prohibited this %{model} from being saved"
```
Comment (sergio-ocon) [inline config/locales/en.yml]: Didn't you fix this in the past? 
Have you rebased?

**Pull request merged**